apps/myjobs/README.md

```diff
@@ -1,141 +1,30 @@
 # Job Composer (renamed from My Jobs)
 
-[![GitHub version](https://badge.fury.io/gh/OSC%2Food-myjobs.svg)](https://badge.fury.io/gh/OSC%2Food-myjobs)
+This app is a Rails app for Open OnDemandfor creating and managing batch jobs
```

Comment:
Space was missing and the wording might be cleaner as something like:
```
This is a Rails app in Open OnDemand for creating aand managing batch jobs...
```

Reply:
make sure you change aand to and

Reply:
Not sure what happened to Travis's remark, but this is what he had.

"This is a Rails app in Open OnDemand for creating and managing batch jobs..."

Reply:
```suggestion
This app is a Rails app we ship with Open OnDemand that used to create and manage batch jobs
```